### PR TITLE
Copy edit suggestions and questions

### PR DESCRIPTION
**Pull request by lamagnifica**
@cpennington please feel free to reject any of these suggestions. I did have a few questions for you, marked "Cale" as comments.

### CODE REVIEW

File: oep-template.rst [hunk 1]
```diff
@@ -33,80 +33,86 @@ OEP-XXX: OEP Template
...
 =============
 
-The technical specification should describe the syntax and semantics of any new API,
-or the details of what the Best Practice, Process, or Architecture being proposed
-by the OEP are.
+The specification describes the technical details of the Architecture, Best
+Practice, or Process change proposed by the OEP. If the proposal includes a new
+API, specify its syntax and semantics.
 
+.. Cale, I've rewritten this guessing that "API" does not need to be added to the "Type" values in the table, and that an API will typically just come along with one of the other types.
```
Comment (cpennington) [inline oep-template.rst]: Yup, makes sense.

File: oeps/oep-0001.rst [hunk 2]
```diff
@@ -2,186 +2,211 @@
...
+development workflow with a patch submission.
+
+Each OEP must have a Author: someone who writes the OEP using the style and
+format described below, shepherds the discussions in the appropriate forums,
+and attempts to build community consensus around the idea. The OEP Author
+should first attempt to ascertain whether the idea is appropriate for an OEP.
+Posting to the `edx-code`_ mailing list, ``arch@edx.org`` and
+``archexternal@edx.org`` is the best way to go about this.
+
+ .. Cale, it's unclear if they should contact all three of these ^^ or some subset.
```
Comment (cpennington) [inline oeps/oep-0001.rst]: Hrm... Ok, I'll try and clarify.

File: oeps/oep-0001.rst [hunk 3]
```diff
@@ -2,186 +2,211 @@
...
 
-When a OEP is Accepted, Rejected or Withdrawn, the OEP should be updated
+When an OEP is Accepted, Rejected, or Withdrawn, the OEP should be updated
 accordingly. In addition to updating the status field, at the very least the
-Resolution header should be added with a link to the relevant post in the edx-code
-mailing list archive or to the appropriate section of the PR.
+Resolution header should be added with a link to the relevant post in the
+edx-code mailing list archive or to the appropriate section of the PR.
+
+.. Cale, the template does not have a "Resolution header". Update the Status and Last Modified instead, maybe?
```
Comment (cpennington) [inline oeps/oep-0001.rst]: Yeah, it should be status.

**Comment by cpennington** [comment]
@lamagnifica: Can you review the commit I added to this PR and see if there's any further edits you'd like to make?

File: oep-template.rst [hunk 4]
```diff
@@ -25,6 +25,9 @@ OEP-XXX: OEP Template
 +---------------+-------------------------------------------+
 |  Created      | <date created on, in YYYY-MM-DD format>   |
 +---------------+-------------------------------------------+
+| `Resolution`  | <links to any discussions where the final |
+|               | status was decided.                       |
```
Comment (lamagnifica) [inline oep-template.rst]: I missed the close angle bracket! >

File: oeps/oep-0001.rst [hunk 5]
```diff
@@ -216,120 +241,131 @@ Each OEP should have the following parts:
...
+|               | Rejected | Withdrawn | Final |            |
+|               | Replaced>                                 |
++---------------+-------------------------------------------+
+| Type          | <Architecture | Best Practice |           |
+|               | Process>                                  |
++---------------+-------------------------------------------+
+| Created       | <date created on, in YYYY-MM-DD format>   |
++---------------+-------------------------------------------+
+| `Resolution`  | <links to any discussions where the final |
+|               | status was decided.                       |
```
Comment (lamagnifica) [inline oeps/oep-0001.rst]: here too :(

**Comment by lamagnifica** [comment]
your commit looks terrific, @cpennington 
Adding a new commit now for my two missing angle brackets...

File: oep-template.rst (hunk 4 above)
Comment (cpennington) [inline oep-template.rst]: That wasn't you, that was me. Thanks for catching it. =)

**Comment by lamagnifica** [comment]
👍 merge or give me the go ahead @cpennington !

**Comment by cpennington** [comment]
:+1:. I'll squash and merge.

**Pull request merged**